src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -356,12 +351,16 @@ private async Task<List<Rule>> DeleteAllRulesInPolicy(RequestToDelete policyToDe
 
                 existingDelegationPolicy.Rules.Clear();
 
-                // Write delegation policy to blob storage
-                MemoryStream dataStream = PolicyHelper.GetXmlMemoryStreamFromXacmlPolicy(existingDelegationPolicy);
-                Response<BlobContentInfo> response = await _policyRepository.WritePolicyConditionallyAsync(policyPath, dataStream, leaseId);
-                if (response.GetRawResponse().Status != (int)HttpStatusCode.Created)
+                Response<BlobContentInfo> response;
+                try
+                {
+                    // Write delegation policy to blob storage
```

Comment:
This comment assumes that it knows what the current implementation of the IPolicyRepository interface use as persistence technology. I've seen this in a few other places as well.

If we do decide to keep a comment here if should be about data being stored "somewhere".
```suggestion
                    // Persisting the delegation policy to storage.
```